build-static.sh

```diff
@@ -137,7 +137,14 @@ else
 	${spcCommand} build --debug --enable-zts --build-embed ${extraOpts} "${PHP_EXTENSIONS}" --with-libs="${PHP_EXTENSION_LIBS}"
 fi
 
-if ! type "xcaddy" >/dev/null 2>&1; then
+if ! type "go" >/dev/null 2>&1; then
+	echo "The \"go\" command must be installed. We strongly recommended installed Homebrew for automatically install Golang"
```

Comment:
```suggestion
	echo "The \"go\" command must be installed."
```

Not really on Linux. Any Go installation is fine as long as it is recent enough. For instance we use the Docker version by default.